.coveragerc

```diff
@@ -0,0 +1,7 @@
+[report]
+omit =
+    */site-packages/*
+    */pypy-*/*
+    .tox/*
+    tests/*
```

Comment:
There is no need for test code that does not get executed; so we don't want to omit that.